src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy

```diff
@@ -218,156 +216,49 @@ mode with existing flows and hold flows of different table-mode types"() {
-        verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-            rules.find { it.cookie == flowInfoFromDb2.forwardPath.cookie.value }.tableId == SINGLE_TABLE_ID
-            rules.find { it.cookie == flowInfoFromDb2.reversePath.cookie.value }.tableId == SINGLE_TABLE_ID
-            //shared rules should be deleted, issue #3546
-            rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-        }
-
-        and: "Flow is valid and UP"
-        northbound.validateFlow(flow.flowId).each { direction -> assert direction.asExpected }
-        wait(WAIT_OFFSET / 2) { assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP }
-
-        when: "Update the flow"
-        flowHelperV2.updateFlow(flow.flowId, flow.tap { it.description = it.description + " updated" })
+        and: "#eventName"
+        event(flow)
 
-        then: "Flow rules on the switch are recreated in multi table mode"
-        def flowInfoFromDb3
+        then: "Flow rules go to multi table mode"
         wait(RULES_INSTALLATION_TIME) {
-            flowInfoFromDb3 = database.getFlow(flow.flowId)
-            verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-                rules.find { it.cookie == flowInfoFromDb3.forwardPath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-                rules.find { it.cookie == flowInfoFromDb3.reversePath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-                rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-            }
+            "flow rules are in tables"(sw, flow) == [INGRESS_RULE_MULTI_TABLE_ID]
         }
-
-        when: "Update switch properties(multi_table: false) on the switch"
-        northbound.updateSwitchProperties(sw.dpId, changeSwitchPropsMultiTableValue(initSwProps, false))
-
-        and: "Reroute(intentional) the flow via APIv1"
-        with(northbound.rerouteFlow(flow.flowId)) { !it.rerouted }
-        wait(rerouteDelay + WAIT_OFFSET) {
-            assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.UP
-        }
-
-        then: "Flow rules on the switch are not recreated in single table mode because flow wasn't rerouted"
-        verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-            rules.find { it.cookie == flowInfoFromDb3.forwardPath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-            rules.find { it.cookie == flowInfoFromDb3.reversePath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-            rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-        }
-
-        when: "Reroute(intentional) the flow via APIv2"
-        with(northboundV2.rerouteFlow(flow.flowId)) { !it.rerouted }
-        wait(rerouteDelay + WAIT_OFFSET) {
-            assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.UP
-        }
```

Comment:
Even if rerouting of one switch flow has no sense we must be sure that this operation wouldn't crash the system and that response is valid. So we should not remove this test case

Reply:
Do we need to test this on environment for 2 minutes or this can be verified on unit tests level?

Reply:
We can't verify it in unit tests because in positive scenario it requires 2 components: north bound, flowhs topology, in negative scenario in requires 3 components: north bound, flowhs topology, floodlight

Reply:
![Will be done](https://www.meme-arsenal.com/memes/b0844e1be8d8b71b03900331a30534db.jpg)

Reply:
Done